index.js

```diff
@@ -571,6 +582,25 @@ function issecure(req, trustProxy) {
 }
 
 /**
+ * Create a new Cookie for this request.
+ *
+ * @param {Object} cookieOptions
+ * @param {Object} req
+ * @param {Boolean} [trustProxy]
+ * @return {Cookie}
+ * @private
+ */
+function createCookie(cookieOptions, req, trustProxy) {
+  cookieOptions = cookieOptions || {};
```

Comment:
Please don't re-assign an argument variable, as v8 will never optimize the function.